WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -71,8 +71,29 @@ class StoreStatsV4PeriodViewController: UIViewController {
     @IBOutlet private weak var timeRangeBarBottomBorderView: UIView!
 
     private var lastUpdatedDate: Date?
-    private var yAxisMinimum: String = Constants.chartYAxisMinimum.humanReadableString()
-    private var yAxisMaximum: String = ""
+
+    private var currencyCode: String {
+        return CurrencySettings.shared.symbol(from: CurrencySettings.shared.currencyCode)
+    }
+
+    private var revenueItems: [Double] {
+        return orderStatsIntervals.map({ ($0.revenueValue as NSDecimalNumber).doubleValue })
+    }
+
+    private var yAxisMinimum: String {
+        let min = revenueItems.min() ?? 0
+        return CurrencyFormatter().formatHumanReadableAmount(String(min),
+                                                             with: currencyCode,
+                                                             roundSmallNumbers: false) ?? String()
+    }
+
+    private var yAxisMaximum: String {
+        let max = revenueItems.max() ?? 0
+        return CurrencyFormatter().formatHumanReadableAmount(String(max),
+                                                             with: currencyCode,
+                                                             roundSmallNumbers: false) ?? String()
+    }
```

Comment:
I think we should group these together with the `xAxisMinimum` and `xAxisMaximum` properties.